src/components/ImportRecordsFlow/stepThree/__test__/assignFieldModalTitle.spec.js

```diff
@@ -5,8 +5,7 @@ import AssignFieldModalTitle from '../assignFieldModalTitle';
 describe('<assignFieldModalTitle />', () => {
     it('should render the right modal description', () => {
         const field = 'test';
-        const component = mount(<AssignFieldModalTitle field={field} />);
-        const compareText = `Assign to the field: ${field}`;
-        expect(component.text()).toBe(compareText);
+        const component = mount(<AssignFieldModalTitle field="test" />);
+        expect(component.text()).toBe(`Assign to the field: ${field}`);
```

Comment:
in prev comment here I meant to not use `${field}`:
expect(component.text()).toBe('Assign to the field: test');